include/open62541/server_pubsub.h

```diff
@@ -314,6 +315,7 @@ typedef struct {
 typedef struct {
     UA_String name;
     UA_PublishedDataSetType publishedDataSetType;
+    UA_Boolean sendViaWriterGroupTopic;
```

Comment:
Maybe a more precise name of this flag would help to understand it. In addition add a short inline comment.

Reply:
Renamed it to batchMessagesViaWriterGroupTopic and added the coment.